### ISSUE TITLE
想请教一下为什么会报错

### ISSUE THREAD

**Issue by franky519** [smthemex/ComfyUI_KV_Edit#5]
![Image](https://github.com/user-attachments/assets/c276202f-6aaf-41a8-8a1f-c41f06e7e500)

想请教一下为什么会报错，基本上是直接把仓库里的示例工作流拖进去的

报错信息：

Error Details
Node ID: 19

Node Type: KV_Edit_Sampler

Exception Type: RuntimeError

Exception Message: The expanded size of the tensor (3644) must match the existing size (3525) at non-singleton dimension 3.  Target sizes: [1, 24, 1104, 3644].  Tensor sizes: [1, 1, 3525]

Stack Trace
File "/root/ComfyUI/execution.py", line 327, in execute
    output_data, output_ui, has_subgraph = get_output_data(obj, input_data_all, execution_block_cb=execution_block_cb, pre_execute_cb=pre_execute_cb)

  File "/root/ComfyUI/execution.py", line 202, in get_output_data
    return_values = _map_node_over_list(obj, input_data_all, obj.FUNCTION, allow_interrupt=True, execution_block_cb=execution_block_cb, pre_execute_cb=pre_execute_cb)

  File "/root/ComfyUI/execution.py", line 174, in _map_node_over_list
    process_inputs(input_dict, i)

  File "/root/ComfyUI/execution.py", line 163, in process_inputs
    results.append(getattr(obj, func)(**inputs))

  File "/root/ComfyUI/custom_nodes/ComfyUI_KV_Edit/kv_edit_node.py", line 195, in main
    output_latent=pipeline.edit(model_int,opts,inp,inp_target,mask)

  File "/root/miniconda3/lib/python3.10/site-packages/torch/utils/_contextlib.py", line 116, in decorate_context
    return func(*args, **kwargs)

  File "/root/ComfyUI/custom_nodes/ComfyUI_KV_Edit/gradio_kv_edit_inf.py", line 45, in edit
    x = self.model(inp, inp_target, mask, opts)

  File "/root/miniconda3/lib/python3.10/site-packages/torch/nn/modules/module.py", line 1736, in _wrapped_call_impl
    return self._call_impl(*args, **kwargs)

  File "/root/miniconda3/lib/python3.10/site-packages/torch/nn/modules/module.py", line 1747, in _call_impl
    return forward_call(*args, **kwargs)

  File "/root/miniconda3/lib/python3.10/site-packages/torch/utils/_contextlib.py", line 116, in decorate_context
    return func(*args, **kwargs)

  File "/root/ComfyUI/custom_nodes/ComfyUI_KV_Edit/flux/kv_edit.py", line 148, in forward
    z_fe, info = denoise_kv_inf(self.model, img=inp["img"], img_ids=inp['img_ids'],

  File "/root/ComfyUI/custom_nodes/ComfyUI_KV_Edit/flux/sampling.py", line 278, in denoise_kv_inf
    v_tar = model(

  File "/root/miniconda3/lib/python3.10/site-packages/torch/nn/modules/module.py", line 1736, in _wrapped_call_impl
    return self._call_impl(*args, **kwargs)

  File "/root/miniconda3/lib/python3.10/site-packages/torch/nn/modules/module.py", line 1747, in _call_impl
    return forward_call(*args, **kwargs)

  File "/root/ComfyUI/custom_nodes/ComfyUI_KV_Edit/flux/model.py", line 157, in forward
    img, txt = block(img=img, txt=txt, vec=vec, pe=pe, info=info)

  File "/root/miniconda3/lib/python3.10/site-packages/torch/nn/modules/module.py", line 1736, in _wrapped_call_impl
    return self._call_impl(*args, **kwargs)

  File "/root/miniconda3/lib/python3.10/site-packages/torch/nn/modules/module.py", line 1747, in _call_impl
    return forward_call(*args, **kwargs)

  File "/root/ComfyUI/custom_nodes/ComfyUI_KV_Edit/flux/modules/layers.py", line 321, in forward
    attn = attention(q, k, v, pe=pe, pe_q = info['pe_mask'],attention_mask=info['attention_scale'])

  File "/root/ComfyUI/custom_nodes/ComfyUI_KV_Edit/flux/math.py", line 14, in attention
    x = torch.nn.functional.scaled_dot_product_attention(q, k, v,attn_mask=attention_mask)

**Comment by smthemex** [comment]
请用示例的模型，而非微调的

**Comment by franky519** [comment]
> 请用示例的模型，而非微调的

我这个也不是微调的模型？ flux-1-dev.sft  ？ fp8 的支持吗

**Comment by smthemex** [comment]
目前不支持fp8直接加载，还有你的工作流不要用其他的mask节点，可能其他插件的mask节点输出的格式不同。
当然，看报错是向量形状不匹配，建议用官方的单体模型测试
